_data/people/fatala22.yml

```
@@ -6,12 +6,27 @@ institution: University of Puerto Rico at Mayaguez
 website: http://charma.uprm.edu/~malik/
 photo: /assets/images/team/Sudhir-Malik.png
 presentations:
+  - title: "Software Sustainability Core"
+  date: 2019-09-03
```

```suggestion
    date: 2019-09-03
```

Indention in YAML is arbitrary, but must be consistent.